src/components/OperationTag.js

```diff
@@ -43,8 +43,10 @@ export default class OperationTag extends React.Component {
     return (
       <div
         className={showTag ? "opblock-tag-section is-open" : "opblock-tag-section"}
-      >
-        <h1
+        >
+        <div
+          role="rowheader"
```

Comment:
```suggestion
          role="button"
```

I think it makes more sense to make it a `button` - since this isn't a table, we shouldn't use `rowheader`

Reply:
Hm, I don't think a `button` would be semantically correct either... since there's a possibility of having a link inside. Not sure what the best solution for that is.

Reply:
Yeah, that's kinda my trouble, too :(. It's both a container for content as well as a button that triggers things.

Based on our discussion, I think the right move is to refactor how the content is rendered. [this demo from w3](https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion) seems to be a comparable structure, we just need to shift things up a little to be compliant that way.

That way `button` doesn't contain content, but it is associated.

Thoughts?